napari/utils/action_manager.py

```diff
@@ -247,8 +247,14 @@ def _update_gui_elements(self, name: str):
 
         # update buttons with shortcut and description
         if name in self._shortcuts:
-            shortcut = self._shortcuts[name]
-            shortcut_str = f' ({Shortcut(shortcut).platform})'
+            shortcuts = self._shortcuts[name]
+            shortcut_str = (
+                '('
+                + ','.join(
```

Comment:
If a shortcut uses `,` it might look confusing?

`Ctrl+,,Ctrl+[`. / `Ctrl-,,Ctrl-[`

Maybe using a space?

Reply:
I'm afraid space might be too subtle to be seen, I tried | but it looked weird. I can use `trans._('or')`, with space on each sides assuming we'll rarely get more than 2 ?